Formula/khal.rb

```diff
@@ -64,46 +74,74 @@ class Khal < Formula
   end
 
   resource "vdirsyncer" do
```

Comment:
vdirsyncer exists in hombrew (it is both an app and a lib), would it be an option to use depends_on rather than vendoring all those deps? If so, any hint on how that works? I couldn't get it to work...